packages/shared/lib/clients/sync.client.ts

```diff
@@ -30,8 +30,8 @@ import telemetry, { LogTypes } from '../utils/telemetry.js';
 import errorManager, { ErrorSourceEnum } from '../utils/error.manager.js';
 import { NangoError } from '../utils/error.js';
 import type { RunnerOutput } from '../models/Runner.js';
-import { isTest, isProd, getLogger, metrics } from '@nangohq/utils';
-import { isErr, resultOk, type Result, resultErr } from '@nangohq/utils';
+import type { LogContext, LogContextGetter } from '@nangohq/logs';
```

Comment:
so is it ok to still depends on `logs` just for types? It doesn't load anything else from `logs`?

Reply:
that's what I'm trying to figure out, in theory it should be okay because we don't ship devDeps

Reply:
Seems to work if we trust the CI 🙌🏻